ports/raspberrypi/Makefile

```diff
@@ -270,9 +270,15 @@ $(BUILD)/boot2.bin: $(BUILD)/boot2.elf
 	$(STEPECHO) "OBJCOPY $<"
 	$(Q)$(OBJCOPY) -O binary $< $@
 
-$(BUILD)/boot2.elf: $(BOOT2_S_UPPER) | $(BUILD)/
```

Comment:
`BOOT2_S_UPPER ?= sdk/src/rp2_common/boot_stage2/boot2_generic_03h.S` and related lines should be removed unless it's possible/intended for boards to go back to an asm boot2 in which case removing this line is probably wrong

Reply:
(other lines being the BOOT2_S_UPPER settings in individual board files)